products/memgraph/upcoming/tutorials/wiki-articles.md

```diff
@@ -0,0 +1,182 @@
+
+# Introduction
+
+A free, multilingual, widely available subset of collected world's knowledge. The first Wikipedia edit happened in 2001 which started the avalanche of written Wikipedia articles. There were more than 6,200,000 English Wikipedia articles in 2020. Wikipedia is a common starting point for general information, backed up by reliable sources outside of the site. Quality of articles is maintained by various editors, which anyone can become.
+
+In this demo, you will explore the contents and contexts of 339 featured Wikipedia articles.
+
+The following datasets (reduced) were used to create this demo:
+1. [Wikipedia network of top categories](https://snap.stanford.edu/data/wiki-topcats.html)
+2. [Linked WikiText-2](https://rloganiv.github.io/linked-wikitext-2/#/)
+# Data model
+Wikipedia Articles is a [graph](https://en.wikipedia.org/wiki/Graph_theory#:~:text=In%20mathematics%2C%20graph%20theory%20is,also%20called%20links%20or%20lines) dataset consisting of the [nodes](https://en.wikipedia.org/wiki/Vertex_(graph_theory)) and [edges](https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms#edge).
+
+**Nodes** can be of the following types:
+
+1. `Article` with a property `name`
+2. `Term`  with a property `name`
+3. `Category` with a property `name`
+
+examples:
+- `node:Article { name: 'Protein' }`
+- `node:Term { name: 'protein folding' }`
+- `node:Category { name: 'Molecular biology' }` 
+
+**Edges** can be of the following types:
+
+1. `:CONTAINS` with a property `count` - connects `Article` to `Term`
+2. `:PARENT_TO` - connects `Category` to `Article
+
+examples:
+- `(n:Article { name: 'Protein' })` `-[:CONTAINS { count: 3 }]->` `(n:Term { name: 'protein folding' })`
+- `(n:Category { name: 'Molecular biology' })` `-[:PARENT_TO]->` `(n:Article { name: 'Protein' }`
+
+
+What's an Article made of? Terms!
+>Wikipedia `Articles` contain many words and phrases called `Terms`. This relation is represented with an edge `:CONTAINS`. If `Article`'s text contains a `Term` they are connected with an edge `:CONTAINS` (from `Article` to `Term`). Different `Article` nodes can be connected to the same `Term` node. `count` property on an edge `:CONTAINS` is a number of repeating `Term` in an `Article`.
+
+Who are Article's parents? Categories!
+>Wikipedia `Category` is a parent of many `Articles`. This relation is represented with an edge `:PARENT_OF`. If `Category` is a parent of an `Article` they are connected with an edge `:PARENT_OF` (from  `Category` to `Article`). An `Article` can have multiple parent `Categories`.
+
+
```

Comment:
Add a small text saying what is in the picture below, that is a representation of the graph model.